.github/workflows/ui-tests.yml

```diff
@@ -13,7 +13,14 @@ env:
 
 jobs:
   ui-tests:
-    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
Nice! I had not realized that you could use `strategy` in a `workflow_call`-able workflow...